frontend/test/xray/xray.integ.spec.js

```diff
@@ -88,6 +96,21 @@ describe("xray integration tests", () => {
         })
     })
 
+    describe("field x-rays", async () => {
+        it("should render the field x-ray page without errors", async () => {
+            const store = await createTestStore()
+            store.pushPath(`/xray/field/1/approximate`);
+
+            const app = mount(store.getAppContainer());
+            await store.waitForActions(FETCH_XRAY, LOAD_XRAY, { timeout: 20000 })
```

Comment:
you should provide actions in a single array, fix this to
```
await store.waitForActions([FETCH_XRAY, LOAD_XRAY], { timeout: 20000 })
```

Reply:
Same thing also later in the tests. I think we could add a safeguard against this kind of usage (if the second parameter is string / there are more than two parameters, then complain)